api/envoy/service/status/v3/csds.proto

```diff
@@ -124,10 +126,59 @@ message ClientConfig {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.service.status.v2.ClientConfig";
 
+  // GenericXdsConfig is a simpler and generic version of specifying
+  // XDS config status and the dump.
+  // [#next-free-field: 10]
+  message GenericXdsConfig {
+    // Type_url represents the fully qualified name of xDS resource type
+    // like envoy.v3.Cluster, envoy.v3.ClusterLoadAssignment etc.
+    string type_url = 1;
+
+    // Name of the xDS resource
+    string name = 2;
+
+    // This is the :ref:`version_info <envoy_v3_api_field_service.discovery.v3.DiscoveryResponse.version_info>`
+    // in the last processed xDS discovery response. If there are only
+    // static bootstrap listeners, this field will be ""
+    string version_info = 3;
+
+    // The xDS resource config. Actual content depends on the type
+    google.protobuf.Any xds_config = 4;
+
+    // Timestamp when the xDS resource was last updated
+    google.protobuf.Timestamp last_updated = 5;
+
+    // Per xDS resource config status. It is generated by management servers.
+    // It will not be present if the CSDS server is an xDS client.
+    ConfigStatus config_status = 6;
+
+    // Per xDS resource status from the view of a xDS client
+    admin.v3.ClientResourceStatus client_status = 7;
+
+    // Set if the last update failed, cleared after the next successful
+    // update. The *error_state* field contains the rejected version of
+    // this particular resource along with the reason and timestamp. For
+    // successfully updated or acknowledged resource, this field should
+    // be empty.
+    // [#not-implemented-hide:]
+    admin.v3.UpdateFailureState error_state = 8;
+
+    // Is static resource is true if it is specified in the envoy config
```

Comment:
let's maybe remove the mention of Envoy in the comment since this is xDS API, not Envoy specific?

Reply:
Done.